makefiles/deprecated_modules.inc.mk

```diff
@@ -0,0 +1,3 @@
+# Add deprecated modules here
+# Keep this list ALPHABETICALLY SORTED!!!!111elven
+DEPRECATED_MODULES += event_thread_lowest
```

Comment:
What should be used instead?

Reply:
There is no `event_thread_low` or `EVENT_PRIO_LOW`... The only instance of `EVENT_PRIO_LOW` is at

```
drivers/at/at.c
28:#define AT_EVENT_PRIO EVENT_PRIO_LOW
```

I fear this is typo that slipped through Murdock.